docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.md

```diff
@@ -27,7 +27,7 @@ The abstract from the paper is:
 
 <Tip>
 
-To learn how to use SDXL for various tasks, how to optimize performance, and other usage examples, take a look at the [Stable Diffusion XL](/using-diffusers/sdxl) guide.
+To learn how to use SDXL for various tasks, how to optimize performance, and other usage examples, take a look at the [Stable Diffusion XL](../../../using-diffusers/sdxl) guide.
```

Comment:
This worked for me locally with `doc-builder`, but does not work in the PR's documentation endpoint. Should we use absolute links instead? cc @mishig25 in case we are doing something wrong.

There are also broken links from other pages, for example the ones inside the green tip here: https://huggingface.co/docs/diffusers/v0.20.0/en/api/pipelines/attend_and_excite#attendandexcite

Reply:
Its working now from the PR preview! I'll do a quick sweep of the other links to fix any broken ones :)